src/Mouf/Mvc/Splash/Services/SplashUtils.php

```diff
@@ -201,9 +201,31 @@ public static function buildControllerResponse($callback)
         }
 
         $code = http_response_code();
-        $headers = headers_list();
+        $headers = SplashUtils::greatResponseHeaders();
+
+        // Suppress actual headers (re-add by Symfony Response)
+        // If you don't remove old headers, it's duplicated in HTTP Headers
+        foreach ($headers as $key => $head) {
+            header_remove($key);
+        }
```

Comment:
I believe you could simply call `header_remove();` to remove all headers (and therefore skip the foreach loop):
http://php.net/manual/fr/function.header-remove.php

Apart from this very small comment, I'm completely ok with this PR  :+1: